diff --git a/tools/xenstore/xenstored_core.c b/tools/xenstore/xenstored_core.c
index 523b963..c2eb970 100644
--- a/tools/xenstore/xenstored_core.c
+++ b/tools/xenstore/xenstored_core.c
@@ -1901,7 +1901,7 @@ int dom0_domid = 0;
 int dom0_event = 0;
 int priv_domid = 0;
 
-int main(int argc, char *argv[])
+int main_disabled(int argc, char *argv[])
 {
 	int opt, *sock = NULL, *ro_sock = NULL;
 	int sock_pollfd_idx = -1, ro_sock_pollfd_idx = -1;
