diff --git a/tools/libs/toollog/xtl_logger_stdio.c b/tools/libs/toollog/xtl_logger_stdio.c
index 52dfbf5..70f1dc7 100644
--- a/tools/libs/toollog/xtl_logger_stdio.c
+++ b/tools/libs/toollog/xtl_logger_stdio.c
@@ -64,7 +64,7 @@ static void stdiostream_vmessage(xentoollog_logger *logger_in,
             fprintf(lg->f, "%04d-%02d-%02d %02d:%02d:%02d %s ",
                     lt->tm_year+1900, lt->tm_mon+1, lt->tm_mday,
                     lt->tm_hour, lt->tm_min, lt->tm_sec,
-                    tzname[!!lt->tm_isdst]);
+                    _tzname[!!lt->tm_isdst]);
         else
             fprintf(lg->f, "[localtime_r failed: %d] ", errno);
     }
